Modules/EMCAL/include/EMCAL/PedestalChannelCheck.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   PedestalChannelCheck.h
+/// \author Sierra Cantway
+///
+
+#ifndef QC_MODULE_EMCAL_EMCALPEDESTALCHANNELCHECK_H
+#define QC_MODULE_EMCAL_EMCALPEDESTALCHANNELCHECK_H
+
+#include "QualityControl/CheckInterface.h"
+
+namespace o2::quality_control_modules::emcal
+{
+
+/// \brief  Check whether a plot is good or not.
+///
+/// \author Sierra Cantway
+class PedestalChannelCheck : public o2::quality_control::checker::CheckInterface
+{
+ public:
+  /// Default constructor
+  PedestalChannelCheck() = default;
+  /// Destructor
+  ~PedestalChannelCheck() override = default;
+
+  // Override interface
+  void configure() override;
+  Quality check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap) override;
+  void beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult = Quality::Null) override;
+  std::string getAcceptedType() override;
```

Comment:
Would be nice to add some function comments as doxygen comments. They don't need to be long, just to express what the functions are doing. They should be in the form

```c++
/// \brief ...
/// \param firstparam This is the information content of the first param
/// \param secondparam This is the information content of the second param
...
/// \return (for non-void functions)
```